mtp_cashbook/templates/base.html

```diff
@@ -21,10 +21,12 @@
 {% endblock %}
 
 {% block global_subheader %}
+  {% if request.user.is_authenticated %}
   <div class="user-details">
-    {% trans "Logged in as Firstname Lastname" %}
+    {% trans "Logged in as " %}{{ request.user.get_full_name|default:request.user.username }}
```

Comment:
This might be better as:

```
{% blocktrans with username=request.user.get_full_name|default:request.user.username %}
Logged in as {{ username }}.
{% endblocktrans %}
```
… or similar. That way the translation files will have better context for editing.